docs/quickstart.md

```diff
@@ -0,0 +1,23 @@
+# Getting Started
+
+## Logging In
+If you already have a TechPass account, you can use TechPass to sign-in to any of the integrated services on SG TechStack. Simple look for the "***Sign-In with TechPass***" button on the services' sign-in page.
+
+If you have do not have a TechPass account yet, follow the instructions below to sign up for one.
+
+## Signing up for an Account
+
+### Public Officer Users with valid gov.sg email
+If you are a user with a **valid gov.sg email account** and have set up your cloud account, you can start by signing up you TechPass account with your gov.sg email on TechPass portal using your **GSIB device**. 
+
+An invitation email will be sent to your gov.sg mailbox. You will be guided to onboard to TechPass upon accepting the invitation or performing your first sign in to any of the integrated services.  
+Follow the instructions [here](onboard?id=public-officer) for the full onboarding instructions
+
+### Verified Vendors/Contractors working on Gov ICT Projects
+
+You will need to inform your project manager or manager-in-charge. They will work with the corresponding SG TechStack service teams to setup a TechPass account for you.
+
+!> Ensure that you do not have an existing and valid gov.sg email before requesting for a TechPass Account.

```

Comment:
What is the intended message here? Use gov.sg is available?

Reply:
this one hor... need to check with denise/damien